apps/emacs/emacs.talon

```diff
@@ -0,0 +1,316 @@
+app: emacs
+app: Emacs
+-
+tag(): user.tabs
+tag(): user.splits
+tag(): user.line_commands
+
+# ----- GENERAL ----- #
+#suplex: key(ctrl-x)
+exchange: user.emacs("exchange-point-and-mark")
+execute: user.emacs_meta("x")
+execute {user.emacs_command}$: user.emacs(emacs_command)
+execute <user.text>$:
+    user.emacs_meta("x")
+    user.insert_formatted(text, "DASH_SEPARATED")
+evaluate | (evaluate | eval) (exper | expression): user.emacs_meta(":")
+prefix: key(ctrl-u)
+prefix <number_small>: user.emacs_prefix(number_small)
+
+open directory: user.emacs("dired-jump")
+other open directory: user.emacs("dired-jump-other-window")
+fill paragraph: user.emacs("fill-paragraph")
+occurs: user.emacs("occur")
+other scroll [down]: user.emacs_meta("pagedown")
+other scroll up: user.emacs_meta("pageup")
+insert unicode: user.emacs("insert-char")
+abort recursive [edit]: user.emacs("abort-recursive-edit")
+save some buffers: user.emacs("save-some-buffers")
+[toggle] read only [mode] [toggle]: user.emacs("read-only-mode")
```

Comment:
"toggle XXX" seems to be more common in knausj, perhaps remove the other options

Reply:
I heavily use the spoken forms without any `toggle` word, eg. just `line numbers`, `truncate lines`, `debug on error`, `read only`. I'd be okay with making the initial `toggle` mandatory though. Opinion?

Reply:
I've made `[toggle]` only appear at the beginning, not the end. But I've kept the other alternates.

Reply:
I’m fine with that. Thanks!